commands/qc.go

```diff
@@ -320,7 +324,7 @@ func newImportPrivKeyCmd() *cobra.Command {
 		Short: "import priKey ",
 		Example: `
 		importprivkey  ef235aacf90d9f4aadd8c92e4b2562e1d9eb97f0df9ba3b508258739cb013db2 pripassword 
-		importprivkey  5HueCGU8rMjxEXxiPuD5BDku4MkFqeZyd4dZ1jvhTVqvbTLvyTJ pripassword  --format=wif
+		importprivkey  PxBefLecRtTYPoxUUwAbq8m7xGmzDK8gQ71N9qKyhp2j5yN42Rpzc pripassword  --format=wif
```

Comment:
- `-- format wif` is the bitcoin legacy format (https://en.bitcoin.it/wiki/Wallet_import_format) , it means the `qitmeer-wallet` supports to import private key by the bitcoin wif format . 
- `qitmeer-wallet` should support use the itself private key format by default or explicitly by `--format qitmeer`
-  `qitmeer-wallet` should support both  `-- format wif` or `--format qitmeer` and document it clearly using the `help` command

Reply:
QX supports formats beginning with 5, so the format beginning with P is just a format defined by the wallet itself